### PR TITLE
[NL] double height of NL scatter tiles

### PR DESCRIPTION
**Pull request by beets**

### CODE REVIEW

File: static/js/components/subject_page/block.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ import {
   EventTypeSpec,
   TileConfig,
 } from "../../types/subject_page_proto_types";
-import { isNlInterface } from "../../utils/nl_interface_utils";
+import { isNlInterface, isNlInterface } from "../../utils/nl_interface_utils";
```
Comment (juliawu) [inline static/js/components/subject_page/block.tsx:35]: why the double import?

Comment (shifucun) [inline static/js/components/subject_page/block.tsx:35]: dup here?

**Review by shifucun — APPROVED** [review]
thx for update!

**Review by chejennifer — APPROVED** [review]

File: static/js/components/subject_page/block.tsx (hunk 1 above)
Comment (beets) [reply static/js/components/subject_page/block.tsx:35]: v weird! fixed

**Pull request merged**